bundles/org.openhab.binding.haywardomnilogic/src/main/java/org/openhab/binding/haywardomnilogic/internal/HaywardBindingConstants.java

```diff
@@ -0,0 +1,147 @@
+
+/**
+ * The {@link HaywardBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Matt Myers - Initial Contribution
+ */
+@NonNullByDefault
+public class HaywardBindingConstants {
+
+    private static final String BINDING_ID = "haywardomnilogic";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_BACKYARD = new ThingTypeUID(BINDING_ID, "backyard");
+    public static final ThingTypeUID THING_TYPE_BOW = new ThingTypeUID(BINDING_ID, "bow");
+    public static final ThingTypeUID THING_TYPE_BRIDGE = new ThingTypeUID(BINDING_ID, "bridge");
+    public static final ThingTypeUID THING_TYPE_CHLORINATOR = new ThingTypeUID(BINDING_ID, "chlorinator");
+    public static final ThingTypeUID THING_TYPE_COLORLOGIC = new ThingTypeUID(BINDING_ID, "colorlogic");
+    public static final ThingTypeUID THING_TYPE_FILTER = new ThingTypeUID(BINDING_ID, "filter");
+    public static final ThingTypeUID THING_TYPE_HEATER = new ThingTypeUID(BINDING_ID, "heater");
+    public static final ThingTypeUID THING_TYPE_PUMP = new ThingTypeUID(BINDING_ID, "pump");
+    public static final ThingTypeUID THING_TYPE_RELAY = new ThingTypeUID(BINDING_ID, "relay");
+    public static final ThingTypeUID THING_TYPE_SENSOR = new ThingTypeUID(BINDING_ID, "sensor");
+    public static final ThingTypeUID THING_TYPE_VIRTUALHEATER = new ThingTypeUID(BINDING_ID, "virtualHeater");
+
+    public static final Set<ThingTypeUID> BRIDGE_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_BRIDGE);
+
+    public static final Set<ThingTypeUID> THING_TYPES_UIDS = new HashSet<ThingTypeUID>(
+            Arrays.asList(HaywardBindingConstants.THING_TYPE_BACKYARD, HaywardBindingConstants.THING_TYPE_BOW,
+                    HaywardBindingConstants.THING_TYPE_BRIDGE, HaywardBindingConstants.THING_TYPE_CHLORINATOR,
+                    HaywardBindingConstants.THING_TYPE_COLORLOGIC, HaywardBindingConstants.THING_TYPE_FILTER,
+                    HaywardBindingConstants.THING_TYPE_HEATER, HaywardBindingConstants.THING_TYPE_PUMP,
+                    HaywardBindingConstants.THING_TYPE_RELAY, HaywardBindingConstants.THING_TYPE_SENSOR,
+                    HaywardBindingConstants.THING_TYPE_VIRTUALHEATER));
+
+    // List of all Channel ids (bridge)
+    // No Channels
+
+    // List of all Channel ids (backyard)
+    public static final String CHANNEL_BACKYARD_AIRTEMP = "backyardAirTemp";
+    public static final String CHANNEL_BACKYARD_STATUS = "backyardStatus";
+    public static final String CHANNEL_BACKYARD_STATE = "backyardState";
+    public static final String CHANNEL_BACKYARD_ALARM1 = "backyardAlarm1";
+    public static final String CHANNEL_BACKYARD_ALARM2 = "backyardAlarm2";
+    public static final String CHANNEL_BACKYARD_ALARM3 = "backyardAlarm3";
+    public static final String CHANNEL_BACKYARD_ALARM4 = "backyardAlarm4";
+    public static final String CHANNEL_BACKYARD_ALARM5 = "backyardAlarm5";
```

Comment:
Are these used?

Reply:
They are programmatically generated.  I deleted the constants.  Thx.

https://github.com/openhab/openhab-addons/blob/d57d0a19025e25837b033566436cc5d865a7bb54/bundles/org.openhab.binding.haywardomnilogic/src/main/java/org/openhab/binding/haywardomnilogic/internal/handler/HaywardBackyardHandler.java#L109